packages/cli/test/integration/generators/relation.integration.js

```diff
@@ -0,0 +1 @@
+('use strict');
```

Comment:
Empty file, please add more content or remove the file entirely.

Reply:
Done